.github/workflows/codeql.yml

```diff
@@ -0,0 +1,48 @@
+name: "CodeQL"
```

Comment:
how is this different from the default settings we may enable for the  repository?

Reply:
This is maintable . You can also customize it.